music21/musicxml/m21ToXml.py

```diff
@@ -2448,26 +2481,34 @@ def setTitles(self):
         mxScoreHeader = self.xmlRoot
         mxWork = Element('work')
         # TODO: work-number
-        if mdObj.title not in (None, ''):
-            # environLocal.printDebug(['metadataToMx, got title', mdObj.title])
+        firstTitleFound: t.Optional[metadata.Text] = None
+        titles: t.Tuple[metadata.Text, ...] = mdObj['title']
+        if titles:
+            if firstTitleFound is None:
+                firstTitleFound = titles[0]
             mxWorkTitle = SubElement(mxWork, 'work-title')
-            mxWorkTitle.text = str(mdObj.title)
-
+            mxWorkTitle.text = str(titles[0])
         if mxWork:
             mxScoreHeader.append(mxWork)
 
-        if mdObj.movementNumber not in (None, ''):
+        movementNumbers: t.Tuple[metadata.Text, ...] = mdObj['movementNumber']
+        if movementNumbers:
             mxMovementNumber = SubElement(mxScoreHeader, 'movement-number')
-            mxMovementNumber.text = str(mdObj.movementNumber)
+            mxMovementNumber.text = str(movementNumbers[0])
 
         # musicxml often defaults to show only movement title
-        # if no movement title is found, get the .title attr
-        movement_title = ''
-        if mdObj.movementName not in (None, ''):
-            movement_title = str(mdObj.movementName)
-        else:  # it is none
-            if mdObj.title is not None:
-                movement_title = str(mdObj.title)
+        # if no movementName is found in mdObj, set movement title to
+        # the mdObj's first title instead. Fall back to defaults.title if
+        # necessary (and if possible).
+
+        movement_title: str = ''
+
+        movementNames: t.Tuple[metadata.Text, ...] = mdObj['movementName']
+        if movementNames:
+            movement_title = str(movementNames[0])
+        else:  # there are no movementNames
+            if firstTitleFound is not None:
+                movement_title = str(firstTitleFound)
```

Comment:
this is all looking good and validating the choice of `.attribute` vs `.item`

btw -- does the plural work in getitem/setitem?  I think I'll naturally start thinking of this is `mdObj['movementNames']` since that's what we're definitely using as variable names

Reply:
I haven't added hand-crafted plural names to the PropertyDescription.  I talked about it, but I'm wondering if it's worth it.

Reply:
can be later.